src/pages/playground/Playground.tsx

```diff
@@ -230,6 +240,94 @@ export async function handleHash(
   }
 }
 
+export async function resetConfig(
```

Comment:
This function should not be decoding at all. The decoder should have done this already. This function should simply set the decoded state for the playground.